### ISSUE TITLE
MongoDB doesn't work as it should

### ISSUE THREAD

**Issue by InfinityZ25** [lucko/LuckPerms#1927]
Hey, MongoDB works differently than other databases, and whenever I want to use a cluster with a URI it simply won't work. Would you please add URI support instead of just an address?

**Comment by jlsjonas** [comment]
Hi,

I tripped over this too at first but lucko has correctly provided `mongodb_connection_URI` which gets passed directly into the MongoDB driver.

However, I only discovered that after seeing this open issue and going through the code to see if I could create a PR for it 😅 

@lucko I'd recommend adding a section to the documentation, or maybe even just improve the readme description above _connection_ to clarify the uri option is there. As it is very easy to scroll past as surrounding config options are sql-specific.
